packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -35,6 +36,16 @@ const double _kCupertinoFocusColorSaturation = 0.835;
 /// ([CupertinoSwitch] in Flutter) instead, or to find a creative custom
 /// solution.
 ///
+/// {@tool dartpad}
+/// This example shows how you can override the default theme of
```

Comment:
```suggestion
/// This example shows how you can override the default color palette of
```

To avoid confusion, let's not say theme. :)